specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/examples/Library_Create.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "endpoint": "exampleWorkspace.dev.azuresynapse.net",
+    "libraryname": "exampleLibraryName.jar",
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "202": {
```

Comment:
Should we need add all kinds of failure error codes? If yes, then all of the APIs could return 401, 409, 500 as well. Append, flush could return 404 not found and delete could return 204 no content.